### DISCUSSION TITLE
'New Tab' Option in 'Navigate To' Action Not Working

### DISCUSSION THREAD

**Discussion by Abd-Atif** [Budibase/budibase#13536] [Help] [answered]
Hello Budibase Community,

I'm encountering an issue where the 'Navigate To' action is not opening the URL in a new tab, despite having the 'New Tab' option checked. The expectation is that when a user clicks the button, it should open the designated URL in a new tab but it currently remains in the same tab.

Attached is a screenshot for reference:
![image](https://github.com/Budibase/budibase/assets/94058099/d2598780-54f4-4e5d-bef8-a865bfbb611b)

Has anyone faced a similar issue or can provide insights into why this might be happening? Any help or guidance would be greatly appreciated!

Best regards.

**Comment by mikesealey** [comment, marked as answer]
Hi @Abd-Atif 

If you want the destination specified to open in a new tab you should include the full URL. 

When set to "Screen" you can specify the local path `/temp/new/row` which can either open in the current tab, or open in a modal in the current screen according to your settings.

**To open in a new tab**, select URL, and paste in the full URL, with New Tab checked. `https://[specific to you].budibase.app/app/[Your App Name]#/temp/new/row`

Hope this helps.

You can read more in the [documentation.](https://docs.budibase.com/docs/application-actions#navigate-to)
